apisix/plugins/response-rewrite.lua

```diff
@@ -115,6 +152,23 @@ function _M.check_schema(conf)
         end
     end
 
+    if conf.filters then
+        for _, filter in ipairs(conf.filters) do
+            for field, value in pairs(filter) do
```

Comment:
We can use `require` in the schema to do the check.

Reply:
`require` seems not suitable. One of `conf.body` `conf.filters` should specify, so has used `oneOf` in the schema.

Reply:
I mean a `require` for the filter.

Reply:
> @kwanhur I mean a `require` for the filter.

I had placed `minItems = 1` limitation. I found many array type properties also use it.

Reply:
@kwanhur 
Err... I mean using `require` in the filter object, not the filters array.

Reply:
Like https://github.com/apache/apisix/blob/7e1babafd098e26f2648d5af3b7695b3020290db/apisix/schema_def.lua#L341

Reply:
Ok, did `required = {"regex", "replace"}`. Fields `scope` and `options` have default value, so not in `required`.